lib/recurly/BaseClient.js

```diff
@@ -15,20 +15,34 @@ const BINARY_TYPES = [
 ]
 const JSON_CONTENT_TYPE = 'application/json'
 const ALLOWED_OPTIONS = ['body', 'params', 'headers']
+const API_HOSTS = {
+  'us': 'v3.recurly.com',
```

Comment:
Let's make the key's here object keys instead of strings. The two are nearly identical in functionality, but standard keys is a little cleaner